### ISSUE TITLE
Add language information text field to pois

### ISSUE THREAD

**Issue by ztefanie** [digitalfabrik/integreat-cms#2799]
### Motivation
In our target group interviews, we (= target group team) got the input, that end users are insecure about going to pois, because they do not know if the language they know are spoken there. As which language is spoken is very different for every location, we want a text field, and hope the cms users will fill it with, e.g. "on thursday afternoon russian, on other days only german an englisch", or similar.

### Proposed Solution

Add a new text field to the creation / editing form of a poi and return it over the api with the poi.

### Alternatives

We discussed if we just tell the cms users to add this information to the description field, but having this field is a constant reminder and as it is a important feature for the target group, we decided adding this field is the best way to go.

### User Story

As a app user I want know if at a poi my language is spoken so that I know if it is worth for me to go there.
As a cms user, I want to add information about languages spoken at a poi.

**Comment by JoeyStk** [comment]
This is pretty similar to [#2421](https://github.com/digitalfabrik/integreat-cms/issues/2421), which was closed as it was doubtful whether or not this would be properly filled in by the municipalities and would actually be helpful. I suppose it's up to us to do some careful research about it

**Comment by MizukiTemma** [comment]
Another point of discussion was that it can lead to exclusion: for example, when the (mightiest) languages of an user is not listed or the listed language give an impression that the POI is only for certain (ethnic) groups, they may feel not welcomed.

**Comment by MizukiTemma** [comment]
This issue was added to the milestone Q2 so we don't forget to come back to the discussion. Feel free to move it.

**Comment by dkehne** [comment]
#2421 was closed because it was not flexible/adjustable and this issue #2799 is a compromise to enable municipalities to add infos in their own way, if they want. 

from the feedback of the target group it become more significant again..

**Comment by seluianova** [comment]
Draft design 😀
![PXL_20240525_131512718](https://github.com/digitalfabrik/integreat-cms/assets/115008338/37f98736-9ce3-4c5b-9937-ffba7f48689c)

**Comment by nikolahoff** [comment]
@JoeyStk "Decide on the state that a language can take on": state means language always or sometimes spoken, right?

**Comment by JoeyStk** [comment]
> "Decide on the state that a language can take on": state means language always or sometimes spoken, right?

Yes, exactly :)

**Comment by nikolahoff** [comment]
@MizukiTemma @JoeyStk [here](https://www.figma.com/design/6U7R7Xj4wL7sbjxKRmOG9D/CMS-Project?node-id=1996-1126&t=Q6zGabDfAz19jEsq-1) is the design for the language selection.

**Comment by nikolahoff** [comment]
@osmers @JoeyStk the updated design for language selection is [here](https://www.figma.com/design/6U7R7Xj4wL7sbjxKRmOG9D/CMS-Project?node-id=1996-1126&t=0yHPyf9pJYhxKz4A-1)

**Comment by osmers** [comment]
@JoeyStk I'll remove the blocked label then? And unassign Nikola - so someone from the CMS team can grab the issue

**Comment by osmers** [comment]
We could add this issue to the next roadmap (finish till Feb conference) - what do you think?

**Comment by MizukiTemma** [comment]
> We could add this issue to the next roadmap (finish till Feb conference) - what do you think?

Yes 🙆

**Comment by osmers** [comment]
The app team has their corresponding issue blocked by ours but has it planned into their Roadmap till Feb ´25. After talking to @LeandraH it would help if we could maybe split this issue into two
1. providing the API endpoint so they can complete their issue and
2. complete this issue in the CMS (also makes sense that the app completes their issue first so that once municipalities can use it, it also shows up in the app)

Do you think this is feasible?

**Comment by MizukiTemma** [comment]
@osmers @nikolahoff 
Which languages should the selection list contain? All the languages the region have, or all the languages that exists in our system, or including languages that are not supported in Integreat?

**Comment by MizukiTemma** [comment]
@LeandraH 
Is the native name of language enough? Like "Deutsch", "简体中文", "Русский" etc?

Then, we can probably imitate the future API field by adding a list of (random) languages as "spoken_language" to some POIs as sample (for other POIs `"sponek_language": Null`).

**Comment by osmers** [comment]
> @osmers @nikolahoff Which languages should the selection list contain? All the languages the region have, or all the languages that exists in our system, or including languages that are not supported in Integreat?

Puuuh :D maybe for a start all languages that are covered within Integreat bcs it represents most languages for the target group and thus most languages spoken in Beratungs centers? And then we can add languages upon request?

**Comment by LeandraH** [comment]
> @LeandraH Is the native name of language enough? Like "Deutsch", "简体中文", "Русский" etc?
> 
> Then, we can probably imitate the future API fields by adding a list of (random) languages as "always_spoken_language" and "occationally_spoken_language" to some POIs as sample (for other POIs `Null`).

Yes, that sounds great, plus a boolean if the text telling the user to check themselves should be shown :)

**Comment by JoeyStk** [comment]
This is blocked again, this time because of #158

**Comment by osmers** [comment]
Closing for now but can reopen if/when dev capacities become available
